git-protocol/src/fetch/delegate.rs

```diff
@@ -29,12 +29,16 @@ pub trait DelegateBlocking {
     /// Note that some arguments are preset based on typical use, and `features` are preset to maximize options.
     /// The `server` capabilities can be used to see which additional capabilities the server supports as per the handshake which happened prior.
     /// Note that this is called only if we are using protocol version 2.
+    ///
+    /// XXX: `Action::Close` here actually means "skip" for v2. Perhaps that should be a separate
```

Comment:
Agreed, there should be a specialized enum to not 'overload' the meaning of `Close`, which in itself is too specific to really have another meaning.